### DISCUSSION TITLE
brew doctor outputs: "Ruby version 2.6.10 is unsupported on macOS 12…" on macOS 12.6.1

### DISCUSSION THREAD

**Discussion by xyzzy-plugh-plover** [Homebrew/discussions#3823] [Everyday usage] [answered]
`brew doctor` outputs the following after upgrading macOS to 12.6.1:

```
$ brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Ruby version 2.6.10 is unsupported on macOS 12. Homebrew
is developed and tested on Ruby 2.6.8, and may not work correctly
on other Rubies. Patches are accepted as long as they don't cause breakage
on supported Rubies.
```

I assume that ruby 2.6.8 was provided with macOS before 12.6.1, while 12.6.1 now provides 2.6.10.

```
$ whence -p ruby
/usr/bin/ruby

$ /usr/bin/ruby --version
ruby 2.6.10p210 (2022-04-12 revision 67958) [universal.x86_64-darwin21]

$ brew config
HOMEBREW_VERSION: 3.6.6-39-g6a7eac2
ORIGIN: https://github.com/Homebrew/brew.git
HEAD: 6a7eac25e167a1eb2d49e13c8cc530a3188af995
Last commit: 28 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: a8e3cf775e5c0164efdea7306c4071d6ee0f8171
Core tap last commit: 34 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.ZpG9qfXIwk/org.xquartz:0
HOMEBREW_GITHUB_API_TOKEN: set
HOMEBREW_MAKE_JOBS: 4
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: quad-core 64-bit haswell
Clang: 14.0.0 build 1400
Git: 2.38.1 => /usr/local/bin/git
Curl: 7.79.1 => /usr/bin/curl
macOS: 12.6.1-x86_64
CLT: 14.0.0.0.1.1661618636
Xcode: 14.0.1
```

**Comment by Bo98** [comment, marked as answer]
Looks like they have indeed updated Ruby in 12.6.1. Thanks for the info, we'll get the warning fixed for the next brew release. Until then, you can ignore it as there's no breaking changes between the two versions.

**Comment by dani-sanomads** [comment]
Hi, Guys. I'm facing some issues with ruby version and shopify-cli library.
On this command:
> **ruby -rrbconfig -e'p RbConfig::CONFIG["bindir"]'**                            
I'm getting this , i think it is forcing the bundle to use 2.5 instead of 3.2 ruby version.
"/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin"
Can anyone guide?

**Reply by dani-sanomads** [reply]
![Screenshot 2023-02-27 at 9 38 36 PM](https://user-images.githubusercontent.com/101345208/221636700-0e0547ba-09cb-4493-911a-e7f544193cf0.png)
![Screenshot 2023-02-27 at 9 39 07 PM](https://user-images.githubusercontent.com/101345208/221636737-b34ff801-e20d-44b3-a1b1-3a5d2f0361de.png)

**Reply by dani-sanomads** [reply]
<img width="655" alt="Screenshot 2023-02-27 at 10 26 50 PM" src="https://user-images.githubusercontent.com/101345208/221637124-89f56b74-6355-4a7d-b845-18eb4ee373b4.png">
